control-plane/tests/test_cli_inspection.py

```
@@ -3872,6 +3872,61 @@ def test_cli_runtime_visibility_ignores_non_handoff_triage_dispositions(
+        main.main(
+            ["inspect-case-detail", "--case-id", promoted_case.case_id],
+            stdout=stdout,
+            service=service,
+        )
+
+        payload = json.loads(stdout.getvalue())
+        reviews_by_id = {
+            review["action_request_id"]: review for review in payload["action_reviews"]
+        }
+
+        for request in (
```

Ruff (0.15.10)

[error] Import `request` from line 13 shadowed by loop variable

(F402)